test/mnist-pytorch/README.md

```diff
@@ -37,6 +37,11 @@ The baseline CNN is specified in the file 'client/init_model.py'. This script cr
 python init_model.py 
 ```
 
+## Load the dataset
```

Comment:
Can you also add instruction for how to download and stage the data manually (in case auto download breaks again on remote endpoint)?